test/unit/mrd/test_mrd_utils.py

```diff
@@ -45,15 +43,19 @@ def test_featuremap_annotator(tmpdir):
             forward_events += 1
         else:
             reverse_events += 1
-    assert FeatureMapFields.IS_DUPLICATE.value in output_variants.header.info, \
-    f'{FeatureMapFields.IS_DUPLICATE.value} is not in info header {output_variants.header.info}'
+    assert (
+        FeatureMapFields.IS_DUPLICATE.value in output_variants.header.info
+    ), f"{FeatureMapFields.IS_DUPLICATE.value} is not in info header {output_variants.header.info}"
     assert forward_events == 9
     assert reverse_events == 22
     assert total_max_softclip_bases == 81
 
+
 def test_read_signature_ug_mutect():
-    signature = read_signature(pjoin(inputs_dir, "mutect_mrd_signature_test.vcf.gz"))
-    signature_no_sample_name = read_signature(pjoin(inputs_dir, "mutect_mrd_signature_test.no_sample_name.vcf.gz"))
+    signature = read_signature(pjoin(inputs_dir, "mutect_mrd_signature_test.vcf.gz"), return_dataframes=True)
+    signature_no_sample_name = read_signature(
+        pjoin(inputs_dir, "mutect_mrd_signature_test.no_sample_name.vcf.gz"), return_dataframes=True
```

Comment:
can we be more specific here? What do we want to test this dataframe for?
#rows. existing columns, etc.

The cons of doing it like this are:
1. Any change in the function will break the test, and this is not always welcomed.
2. It is hard for the reader to understand what are the requirements from read_signature. 
A unit-test should also serve as a contract, and a  documentation